packages/core/src/App/Containers/AccountSwitcher/account-switcher.jsx

```diff
@@ -599,10 +606,10 @@ class AccountSwitcher extends React.Component {
                             secondary
                         />
                     )}
-                    <div id='dt_logout_button' className='acc-switcher__logout'>
-                        <span className='acc-switcher__logout-text' onClick={this.handleLogout}>
+                    <div id='dt_logout_button' className='acc-switcher__logout' onClick={this.handleLogout}>
+                        <Text color='prominent' size='xs' align='left' className='acc-switcher__logout-text'>
```

Comment:
Why `onClick` is moved to the wrapper component?

Reply:
As per discussion with @zaki-binary, I put it on the container. Is there any negative point about that?

Reply:
@reza-fs , I think the reasoning behind this is so that we don't have to explicitly handle non text oriented props within the component.

So there are two ways to approach this,
i) The same approach as above, use the `onClick` handlers outside of the `<Text />` component with a wrapper
ii) Allow `<Text />` to accept general `props`:
```const Text = ({existing_props, ...props} => {
if (p) return  <p {...props}>{children}</p>;
return <span {...props}>{children}</p>;
};```